tasks/13.txt

```diff
@@ -1,3 +1,16 @@
 Create a SQL query to count current and discontinued products:
 
-| TotalOfCurrentProducts | TotalOfDiscontinuedProducts |
\ No newline at end of file
+| TotalOfCurrentProducts | TotalOfDiscontinuedProducts |
+__________________
+
+var a = db.products.find({Discontinued: 1}).count();
+var b = db.products.find({Discontinued: 0}).count();
+db.products.aggregate([
+    {
+        $project: {
+            _id: 0,
+            "TotalOfCurrentProducts": {$sum: [a, 0]},
+            "TotalOfDiscontinuedProducts": {$sum: [b, 0]},
+        } 
+    }
+])
```

Comment:
last query doesn't do any useful work (db.products.aggregate..) =) Please remove that.
In the current case we can do one query with group by 'Discontinued' or just do 2 calls to DB like you do. Both are correct. Not sure but maybe your solution will be even faster on big DBs.